tests/javascript/form_editor/store/map_form_data_before_saving.spec.js

```diff
@@ -0,0 +1,62 @@
+import { expect } from 'chai';
+import map from '../../../../assets/js/src/form_editor/store/map_form_data_before_saving.jsx';
+
+const data = {
+  id: '1',
+  name: 'My First Form',
+  settings: {
+    segments: ['3'],
+    on_success: 'message',
+    success_message: 'Check your inbox or spam folder to confirm your subscription.',
+    success_page: '5',
+    segments_selected_by: 'admin',
+    placeFormBellowAllPages: true,
```

Comment:
We use underscore or naming properties within the request data. I think we should not mix it.